File: Easy/RemoveElement.rb
nums = [3,2,2,3]
val = 3

def removeElement (nums,val)
    n = 0

    for i in 0...nums.size

        if nums[i] != val
                nums[n] = nums[i]
                n +=1
          end

    end

    puts n

end


removeElement(nums,val)

File: Easy/RemoveElement.js
let nums = [3,2,2,3]
let val = 3

var removeElement = function(nums, val) {
        let n = 0
 
        for(let i=0;i<nums.length ;i++){

            if(nums[i] != val){    
                     
                nums[n] = nums[i]
                n ++             
            }
           
        }   

    console.log(nums)
}


removeElement(nums,val)